src/app/helptext/system/reporting.ts

```diff
@@ -0,0 +1,20 @@
+import { T } from "app/translate-marker";
+
+export const helptext = {
+cpu_in_percentage_placeholder: T('Report CPU usage in percentage'),
+cpu_in_percentage_tooltip: T('Set to display CPU usage as percentages in Reporting.'),
```

Comment:
```suggestion
cpu_in_percentage_tooltip: T('When set, report CPU usage in percent instead of jiffies.'),
```

Reply:
If we are not removing this, just "Report CPU usage in percent instead of jiffies."